### ISSUE TITLE
how to hide or encode the password in catalog properties file

### ISSUE THREAD

**Issue by gomou** [prestodb/presto#14702]
I configure the mysql connector in a catalog properties file like that:
connector.name=mysql
connection-url=jdbc:mysql://example.net:3306
connection-user=root
connection-password=secret
but I hope the "secret"  is encoded or invisible
if I use MD5 to encode this passsword， how presto decode it
thanks

**Comment by mbasmanova** [comment]
I don't believe this is supported out of the box, but this post might be helpful: https://blog.starburstdata.com/technical-blog/starburst-secrets-presto-security
